script.js

```diff
@@ -1,66 +1,81 @@
 let currentValue = '';
 let currentOperator = '';
 let result = 0;
-
-function inputNum(value) {
-  currentValue += value;
-  document.getElementById('output').textContent = currentValue;
-}
-
+let displayScreen = document.getElementById('output');
 let operatorButtons = document.getElementsByClassName('operator');
-function inputOperator(oper) {
+document.querySelector('.cal-body').addEventListener('click', function(event){
+  if(event.target.matches('.key')){
+    let buttonValue = event.target.textContent;
+    if(buttonValue==='+'||buttonValue==='‒'||buttonValue==='×'||buttonValue==='÷'||buttonValue==='%'){
+      operatorClicked(buttonValue);
+    }
+    else if(buttonValue === '='){
+      calculate();
+    }
+    else if(buttonValue === 'AC'){
+      resetValues();
+    }
+    else if(buttonValue ==='+/-'){
+      toggleNegate();
+    }
+    else{
+      numericKeyClicked(buttonValue);
+    }
+  }
+} );
```

Comment:
make sure 
- you add new line space after every function
- add new line before return statements if there is other logic in function